server/controllers/CommentController.js

```diff
@@ -238,6 +238,50 @@ class CommentController {
       comment,
     });
   }
+
+  /**
+  * @description This method deletes a comment on an article
+  * @param {object} req - exporess request object
+  * @param {object} res - express response object
+  * @param {object} next - express next object
+  * @returns {object} returns an object of comment
+  */
+  static async deleteComment(req, res, next) {
+    const userId = req.userData.id;
+    const { commentId } = req.params;
+
+    let comment;
+    try {
+      comment = await Comment.findByPk(commentId, {
+        raw: true,
+      });
+
+      if (!comment) {
+        return res.status(404).json({
+          status: 'failure',
+          message: 'comment not found'
+        });
+      }
```

Comment:
this if block is unnecessary as you already have the `validateResourceId` middleware on this endpoint. It makes this check already that the comment exists.